spec/person_spec.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../person'
+require_relative '../book'
+require_relative '../rental'
+
+RSpec.describe Person do
+  let(:person) { Person.new(18) }
+
+  describe '#initialize' do
+    it 'Return a new Person object' do
+      expect(person).to be_an_instance_of Person
+    end
+
+    it 'Assigns the age' do
+      expect(person.age).to eq 18
+    end
+
+    it 'Assign a default name if no name is given' do
+      expect(person.name).to eq 'Unknown'
+    end
+
+    it 'Assigns parent permission to true by default' do
+      expect(person.instance_variable_get(:@parent_permission)).to eq true
+    end
+
+    it 'Assigns an id between 1 and 1000' do
+      expect(person.id).to be_between(1, 1000)
+    end
+
```

Comment:
- [Optional] To improve readability, I think we should have added a "context" block instead of using so many "it" blocks for the scenario of just creating a new Person object. You may check out this best [testing practices ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/rspec_unit_test.md#unit-testing)

Reply:
Thank you! We implemented your suggestion to use a "context" block instead of multiple "it" blocks for creating a new Person object. 😊

Reply:
We added a "context" block, Thanks!😁